02-basic-programming/README.md

```diff
@@ -28,21 +28,25 @@ Com este curso, você desenvolverá dois programas que têm aplicação em nosso
 1. **Cifra de César**: você criará um programa que criptografa e decodifica mensagens de acordo com o [algoritmo da Cifra de César](https://pt.wikipedia.org/wiki/Cifra_de_C%C3%A9sar) \(_Caesar cipher_\).
 2. **Validação de cartão de crédito**: você fará um programa capaz de verificar se um cartão de crédito é válido de acordo [com o algoritmo de Luhn](https://es.wikipedia.org/wiki/Algoritmo_de_Luhn).
 
-## Syllabus
+## Programa de Estudos
 
-### Unidad 01: [A linha de comandos e o Git](01-shell+git)
+### Unidade 01: [Boas-vindas](https://github.com/cemsbr/curricula-js/tree/ff541dfe3be8d642da32687aed9316c990370fc5/02-basic-programming/00-welcome/README.md)
```

Comment:
@vanessapinheiro asked to remove this unit, so we should remove it from the syllabus.